language-support/ts/daml-ledger/tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "noFallthroughCasesInSwitch": true,
     "outDir": "lib/",
     "module": "commonjs",
+    "esModuleInterop": true,
```

Comment:
This is necessary to make the `events` dependency work since it's not in CommonJS module format.